UndertaleModLib/Models/UndertaleGeneralInfo.cs

```diff
@@ -205,6 +205,21 @@ public enum FunctionClassification : ulong
     /// </summary>
     public UndertaleString Name { get; set; }
 
+    /// <summary>
+    /// Different GameMaker release branches. LTS has some but not all features of equivalent newer versions.
+    /// </summary>
+    public enum BranchType
+    {
+        Unknown,
```

Comment:
Hmm, looking at the other fields, we could just name this to something like `PreLTS2022`

Reply:
I think it can technically remain as this even through the end of deserialization on 2022.0.1... but the same is going to apply for "PostLTS2022" with 2025.0, so fair enough.